CHANGELOG.md

```diff
@@ -4,12 +4,15 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog 1.0.0].
 
+## [Release 9.0.0]
+- Changed `SearchParameters` dataclass field from `q` to `query`
+- Added `search_helpers` module to allow clients to search and process document search responses in one go.
+
 ## [Release 8.0.0]
 - Added `SearchParameters` dataclass for use with search functions using the legacy kwargs from `Client.advanced_search` and new `collections` field for filtering by collections
 - Changed `Client.advanced_search` interface to take in `SearchParameters` as opposed to the legacy kwargs.
 - Added `search_and_decode_response` and `search_judgments_and_decode_response` methods to `Client`
 - Added `SearchResponse`, `SearchResult`, `SearchResultMetadata` classes to encapsulate and process document search responses.
-- Added `search_helpers` module allow clients to search and process document search responses in one go.
```

Comment:
I had initially planned to adjust the 8.0.0 release but realised thats a faff when publishing to public pypi

Reply:
We should probably add a note to the v8 release changelog that it's broken.

Or possibly just say it's a broken release and move all the changes into 9.

Reply:
Would like a `**BREAKING**` on `Client.advanced_search`